### PR TITLE
Google Site search integration

### PR DESCRIPTION
**Pull request by eskfung**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -52,11 +52,28 @@
...
           //]]>
-        </script>
+        </script>-->
         <div class='menu'>
-          <form id='search_form'>
+          <!--<form id='search_form'>
             <input class='st-search-input' id='st-search-input' placeholder='Search the Documentation' type='text' />
+          </form>-->
+
+          <style>
```
Comment (eskfung) [inline _layouts/default.html]: Is it safe to put these styles in our CSS? (Will be loaded on every page)

File: search-results.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Search Results"
+tags: []
+permalink: /search-results/
+portal: university
```
Comment (eskfung) [inline search-results.html]: This option can get removed. It's what dictates the type of sidebar you get, and leaving it out gets you the full sidebar.

File: _layouts/default.html [hunk 3]
```diff
@@ -33,7 +33,7 @@
         </div><!-- /.rightcrumb -->
       </div><!-- /#crumbs -->
       <div class='col side' data-swiftype-index='false'>
-        <div id='st-results-container'></div>
+        <!--<div id='st-results-container'></div>
```
Comment (percyhanna) [inline _layouts/default.html]: Preference is to delete commented-out code instead of leaving it in.

File: _layouts/default.html (hunk 1 above)
Comment (percyhanna) [inline _layouts/default.html]: This `<style>` would not get included in routes that use the [`post`](https://github.com/kissmetrics/km-support/blob/gss-integration/_layouts/post.html) template. Do we want it in both? Does it hurt if it's included in that file? Definitely prefer including this in a global CSS file.

Comment (percyhanna) [inline _layouts/default.html]: If styling is different based on which layout is used, then using a `body.class` or something similar to differentiate between the two layouts would work best.

**Comment by jasoncaldwell** [comment]
@percyhanna @eskfung yeah, the markup isn't finished - i was just demonstrating the functionality.

**Comment by jasoncaldwell** [comment]
@percyhanna @eskfung a couple edits to the branch. lemme know what you're thinking. before we publish we need to:
- Pay for license
- Edit settings slightly on google.com/cse

**Comment by eskfung** [comment]
@jasoncaldwell looks good to me! I removed the Swiftype areas that were commented out; if we want them back we can always revert my commit.

**Comment by percyhanna** [comment]
I'm busy with some other stuff so I hope I'm not holding up this release. Looks good to me.

**Comment by jasoncaldwell** [comment]
@eskfung @percyhanna roger that - thanks for the input. i think we're just waiting on buying a license. hope to do that sometime this week.

**Comment by eskfung** [comment]
Not sure we'll actually need this anymore.

**Pull request closed without merging**